src/matchers/utils.ts

```diff
@@ -10,7 +10,7 @@ const isElementHandle = (value: Handle): value is ElementHandle => {
 
 export const getFrame = async (value: ExpectInputType) => {
   const resolved = await value
-  return isElementHandle(resolved) ? resolved.contentFrame() : resolved
+  return isElementHandle(resolved) ? await resolved.contentFrame() : resolved
```

Comment:
Since this method is async, awaiting isn't needed since it will return a promise that must be awaited anyway.

```suggestion
  return isElementHandle(resolved) ? resolved.contentFrame() : resolved
```